src/network.rs

```diff
@@ -377,9 +381,7 @@ impl<'a> Network<'a> {
 
       let mut app = self.app.lock().await;
       app.playlist_tracks = Some(playlist_tracks);
-      if app.get_current_route().id != RouteId::TrackTable {
-        app.push_navigation_stack(RouteId::TrackTable, ActiveBlock::TrackTable);
-      };
+      app.push_navigation_stack(RouteId::TrackTable, ActiveBlock::TrackTable);
```

Comment:
I believe this was a bug that I noticed while working on this patch, and is technically not related to the core issue.

This conditional push to the navigation stack made it so that:
- On the first playlist navigation, the focus would correctly and as expected, shift to the playlist window/frame.
- If the user were to immediately select another playlist to change focus to, the playlist would render, but the focus would not shift to the playlist.

Technically, it seems like this would happen whenever the user selected a playlist while on a track table.

Reply:
I can't remember why we had the condition there in the first place. Perhaps it was indeed a mistake.